### PR TITLE
Exercise 8

### PR DESCRIPTION
**Pull request by jpatel386**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: .github/workflows/todo_app_ci_pipeline.yml
Comment (JackMeadDev) [inline .github/workflows/todo_app_ci_pipeline.yml:26]: Don't forget to update this when you merge 😄

File: .github/workflows/todo_app_ci_pipeline.yml
Comment (JackMeadDev) [inline .github/workflows/todo_app_ci_pipeline.yml:37]: Thanks for adding the comment, useful for future us!

File: Dockerfile
Comment (JackMeadDev) [inline Dockerfile:63]: Minor, but note that I think this has switched from a default port of 80 for Gunicorn to a default of 5000 - it looks like you don't have this bit documented in your Readme but this would be a good example of where we'd want to catch that we're changing the default behaviour!

File: .github/workflows/todo_app_ci_pipeline.yml
Comment (JackMeadDev) [inline .github/workflows/todo_app_ci_pipeline.yml:43]: I can't see this image in DockerHub but I can see your action pushing to it, so I'm assuming it's just a private image!

**Review by JackMeadDev — APPROVED** [review]
Nice work, I can see your pipeline is all playing ball now, and pushing to both DockerHub and Heroku. It would be good to update your docs to at least record where the app is hosted (particularly since the `HEROKU_APP_NAME` is marked a secret in your pipeline so we can't work it out from there!) but I know you showed me it working on a call so I'm happy with that here. Otherwise I've left a couple of optional suggestions, but feel free to merge this when you're ready

File: .github/workflows/todo_app_ci_pipeline.yml
Comment (jpatel386) [reply .github/workflows/todo_app_ci_pipeline.yml:43]: it is indeed a private image
Screenshot 2022-03-29 at 21 47 36

File: Dockerfile
Comment (jpatel386) [reply Dockerfile:63]: Will add to readme before merge

File: .github/workflows/todo_app_ci_pipeline.yml
Comment (jpatel386) [reply .github/workflows/todo_app_ci_pipeline.yml:26]: will merge and then update on master before I branch again

**Pull request merged**
